lib/run-contentful-export.js

```diff
@@ -22,6 +23,45 @@ export default function runContentfulExport (usageParams) {
     spaceId: clients.source.spaceId
   })
     .then((response) => {
+      if (opts.downloadAssets) {
+        var successCount = 0
+        var warningCount = 0
+        var errorCount = 0
+
+        log.info('Downloading ' + response.assets.length + ' assets')
+
+        var arrayOfDownloadPromises = []
+        response.assets.forEach((asset) => {
+          if (asset.fields.file) {
+            Object.keys(asset.fields.file).forEach((key) => {
+              arrayOfDownloadPromises.push(
+                downloadAsset(asset.fields.file[key].url, opts.exportDir).then(
+                  (downLoadedFile) => {
+                    log.info('-> ' + downLoadedFile)
+                    successCount++
+                  },
+                  (error) => {
+                    log.error('-> error downloading ' + asset.fields.file[key].url + ' => ' + error)
+                    errorCount++
```

Comment:
To ensure that Promise.all always continue execution you can add `Promise.resolve()` here after `errorCount++`